### ISSUE TITLE
i character is displayed strange

### ISSUE THREAD

**Issue by fredkoot** [Shorts1999/ptx-smartmatrix#8]
Hi Sjors,
After some deliberation and testing we decided to use this extension in our kids project(we posted the zigzag issue, 5 months ago). We now have a semi finished product and it works.
However the "i" character is being displayed a bit funny with a trailing "dot".
![image](https://github.com/Shorts1999/pxt-smartmatrix/assets/51207811/a14e014b-9603-4b21-bc85-604ff41383f6)
(size 110x35 cm)

In our testing, kids read this as a "L" instead of "i". Might it be possible to fix this?

By the way: in response to your remark about supporting more configurations, your extension (as it is now)  is good enough. Other extensions take up so much memory in the Microbit, that it is not possible to put other stuff (use the Microbit display for instance) in the code. This is one of the reasons we use this extension. We are very happy with it.

Thanks for providing this extension!
Cheers,
Fred

**Comment by Shorts1999** [comment]
Hello Fred,

I am really glad this library is of use to others, and that it can contribute to teaching about electronics and programming.

I have unfortunately not worked on this library or anything with micro:bit in a very long time. I am very busy in college and with a job besides that as well so I do not have much time to develop on this, I will try to find a moment to fix the issue but I can not make any promises to have this done in the near future, my apologies.

Of course the library is fully open-source (under MIT License) so if you know anyone else who might be able to fix it instead they are more than welcome to do so.

TL;DR I might be able to fix it, but it's been a long time and I don't know when I could have the time to do so. But others might be able to as well.

Kind regards,
Sjors

**Comment by Shorts1999** [comment]
Hello, I have updated the library with a _theoretical_ fix for the wrong letter i. I am unfortunately not able to test this since I do not have any hardware.

Could you try it for me and let me know? No need to rush with it, but if you ever have the time I would really appreciate to know if this has resolved the issue

Cheers,
Sjors

**Comment by fredkoot** [comment]
Wow. You are the best. I just tested it, and it looks good. The dot behind the "i" is gone.
The kids event is tomorrow, so I will be bring it and see their reactions. Thanks for going above and beyond on this.

![Screenshot 2023-06-09 193711](https://github.com/Shorts1999/pxt-smartmatrix/assets/51207811/170cf59c-b530-4003-ba98-4473c773e2b9)
![Screenshot 2023-06-09 193812](https://github.com/Shorts1999/pxt-smartmatrix/assets/51207811/b35694b1-0dd2-480c-aec5-8493fe40df04)
![Screenshot 2023-06-09 193846](https://github.com/Shorts1999/pxt-smartmatrix/assets/51207811/61506ae5-be73-4633-bacb-7757910b6e75)
![Screenshot 2023-06-09 193913](https://github.com/Shorts1999/pxt-smartmatrix/assets/51207811/2436e99f-6d04-4122-965a-c74267a3a8f5)

Cheers,
Fred

p.s. for comparison I put a Microbit on the photo. That gives a better idea about how big this is.

**Comment by fredkoot** [comment]
Not sure if I am supposed to close this, but the "i" bug is gone. So closing the issue.
